app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # before_action :authenticate_user!
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please setup confirmable by adding `:confirmable` to your user model and editing the mail sender and device migration file to make it work as expected when you signup with a new email you should have a confirmation email to continue the registration process as the project requirement mentioned.

![image](https://user-images.githubusercontent.com/68796232/174767779-5f06d26f-c086-4258-89a0-587b0974996d.png)

[Hint]: For more info about setup confirmable see this [video](https://www.youtube.com/watch?v=nun18zM8Gq0).

Reply:
Noted. This will be implemented